.gitignore

```diff
@@ -155,3 +155,6 @@ tmp/
 temp/
 
 # End of https://www.toptal.com/developers/gitignore/api/go,node
+*.egg-info/
+.venv/
+tools/mkdocs-roadmap/site/
```

Comment:
Instead of adding this in the root `.gitignore`, why not add a new `.gitignore` in `tools/mkdocs-roadmap` which can be a bit more specialized?